Dockerfile

```diff
@@ -24,6 +24,7 @@ USER nuto
 
 # create the source directory /home/nuto/source
 RUN mkdir /home/nuto/source
+RUN mkdir /home/nuto/build
 
-# the build directory will be in /home/nuto 
-WORKDIR /home/nuto
+# the build directory will be in /home/nuto/source/build
```

Comment:
Should be `home/nuto/build`. But I think the directory name itself indicates its purpose and we can delete this comment.